app/components/show/controls_component.rb

```diff
@@ -69,57 +69,57 @@ def create_embargo
       return if !item? || embargoed?
 
       render ActionButton.new url: new_item_embargo_path(druid),
-                              label: 'Create embargo',
-                              open_modal: true,
-                              disabled: button_disabled?
+        label: "Create embargo",
+        open_modal: true,
+        disabled: button_disabled?
```

Comment:
Prefer two-line indents in more contexts.

Reply:
💯